data/MI/incentives.json

```diff
@@ -0,0 +1,1400 @@
+    },
+    "start_date": 2023,
+    "end_date": 2023
+  },
+  {
+    "id": "MI-6",
+    "authority_type": "utility",
+    "authority": "mi-consumers-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "mi_consumersEnergy_residentialHeating,CoolingAndWaterHeatingProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 350
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$350 rebate for 21.0 SEER/20 SEER2 or higher ductless mini-split heat pump."
+    },
+    "start_date": 2023,
+    "end_date": 2023
+  },
+  {
+    "id": "MI-7",
+    "authority_type": "utility",
+    "authority": "mi-consumers-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "smart_thermostat",
+    "program": "mi_consumersEnergy_residentialHeating,CoolingAndWaterHeatingProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 10
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$10 rebate for Programmable (Must Replace an Existing Nonprogrammable Thermostat)."
+    },
+    "start_date": 2023,
+    "end_date": 2023
+  },
+  {
+    "id": "MI-8",
+    "authority_type": "utility",
+    "authority": "mi-consumers-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "smart_thermostat",
+    "program": "mi_consumersEnergy_residentialHeating,CoolingAndWaterHeatingProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 50
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$50 rebate for Wi-Fi Enabled (Must Replace an Existing Nonprogrammable Thermostat)."
+    },
+    "start_date": 2023,
+    "end_date": 2023
+  },
+  {
+    "id": "MI-9",
+    "authority_type": "utility",
+    "authority": "mi-consumers-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "smart_thermostat",
+    "program": "mi_consumersEnergy_residentialHeating,CoolingAndWaterHeatingProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 100
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$100 rebate for Wi-Fi Enabled (Must Replace an Existing Nonprogrammable Thermostat)."
+    },
+    "start_date": 2023,
+    "end_date": 2023
+  },
+  {
+    "id": "MI-10",
+    "authority_type": "utility",
+    "authority": "mi-consumers-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "mi_consumersEnergy_windowsAndInsulationRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 125,
+      "minimum": 50,
+      "maximum": 125
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Up to $125 rebate for Roof (Attic) Insulation R Value of 48 or less (must insulate a minimum of 500 square feet of attic area)."
```

Comment:
The capitalized words here and in the next several incentives don't need to be capitalized